src/MSBuild.Abstractions/MSBuildConversionWorkspace.cs

```diff
@@ -34,7 +34,37 @@ public MSBuildConversionWorkspace(ImmutableArray<string> paths, bool noBackup)
                 }
 
                 var root = new MSBuildProjectRootElement(ProjectRootElement.Open(path, collection, preserveFormatting: true));
-                if (IsSupportedProjectType(root))
+                // maybe do it here to root?
+                if (winUI3)
+                {
+                    // remove from root improrts and import groups...
+                    foreach (ProjectImportElement i in root.Imports)
+                    {
+                        if (i.Project.EndsWith("Xaml.CSharp.targets"))
```

Comment:
a comment describing why you are doing this would be helpful

Reply:
done